### PR TITLE
General update

### PR DESCRIPTION
**Pull request by bztd**

### CODE REVIEW

**Comment by bztd** [comment]
@hfiguiere  Could you please review this request to be merged? It works more or less with spotify.

File: com.github.muriloventuroso.givemelyrics.json [hunk 1]
```diff
@@ -17,9 +17,25 @@
         "--metadata=X-DConf=migrate-path=/com/github/muriloventuroso/givemelyrics/"
     ],
     "modules": [
+        {
```
Comment (hfiguiere) [inline com.github.muriloventuroso.givemelyrics.json]: this should be using the `shared-modules`.

File: com.github.muriloventuroso.givemelyrics.json [hunk 2]
```diff
@@ -17,9 +17,25 @@
...
+                }
+            ]
+        },
         {
             "name": "givemelyrics",
             "buildsystem": "meson",
+            "post-install": [
+                "install -Dm644 /app/share/icons/hicolor/24x24@2/apps/com.github.muriloventuroso.givemelyrics.svg /app/share/icons/hicolor/scalable/apps/com.github.muriloventuroso.givemelyrics.svg",
+                "mv /app/share/icons/hicolor/24x24/actions/* /app/share/icons/hicolor/scalable/actions/",
+                "rm -r /app/share/icons/hicolor/[0-9]*"
```
Comment (hfiguiere) [inline com.github.muriloventuroso.givemelyrics.json:27]: why do you remove these?

File: com.github.muriloventuroso.givemelyrics.json [hunk 3]
```diff
@@ -29,6 +45,18 @@
...
                     "path": "eos-theme.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "meson.build.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "LyricsFetcher.vala.patch"
+                },
```
Comment (hfiguiere) [inline com.github.muriloventuroso.givemelyrics.json:56]:
```suggestion
                    "paths": [
                        "eos-theme.patch",
                        "meson.build.patch",
                        "LyricsFetcher.vala.patch"
                    ]
                },
```

File: com.github.muriloventuroso.givemelyrics.json [hunk 4]
```diff
@@ -29,6 +45,18 @@
...
+                {
+                    "type": "patch",
+                    "path": "com.github.muriloventuroso.givemelyrics.appdata.xml.in.patch"
```
Comment (hfiguiere) [inline com.github.muriloventuroso.givemelyrics.json]:
```suggestion
                    "paths": [
                        "eos-theme.patch",
                        "meson.build.patch",
                        "LyricsFetcher.vala.patch",
                        "com.github.muriloventuroso.givemelyrics.appdata.xml.in.patch"
                    ] 
```

File: com.github.muriloventuroso.givemelyrics.json (hunk 2 above)
Comment (bztd) [reply com.github.muriloventuroso.givemelyrics.json:27]: For the error non-png-icon-in-hicolor-size-folder. There are .svg in nxn folders.
https://buildbot.flathub.org/#/builders/6/builds/143748

**Pull request merged**